roles/wazuh/ansible-filebeat-oss/templates/elasticsearch.yml.j2

```diff
@@ -162,6 +162,7 @@
       "data.dstip",
```

Comment:
This template should not exists, add the dynamic fetch for OpenDistro also please

Reply:
Done on 600dd40896cdec05c9211731e45b7987537f1c4a.